README.txt

```diff
@@ -962,6 +962,20 @@ users. root can enable this for regular users as well by:
  2. "/etc/init.d/openibd restart"
 
 
+* CAP_NET_RAW and root access
+
+ VMA_WARNING: ******************************************************************************
```

Comment:
Lines switched order + add <interface name>
Content is subject to technical review
```
VMA_WARNING: * Interface <Interface name> will not be offloaded
VMA_WARNING: * Usage is restricted to root or CAP_NET_RAW privileges.                     *
```